source/_integrations/ebusd.markdown

```diff
@@ -13,6 +13,8 @@ ha_integration_type: integration
 
 Integration between [ebusd](https://github.com/john30/ebusd/) daemon for communication with eBUS heating systems, and Home Assistant using sensor component.
 
+Note: [ebusd](https://github.com/john30/ebusd/) also supports a more generic approach using the [MQTT discovery feature](https://www.home-assistant.io/docs/mqtt/discovery/), which allows to make almost everything available as entities in home assistant automatically, see [documentation and setup instructions here](https://github.com/john30/ebusd/wiki/MQTT-integration).
```

Comment:
This page is about the ebusd integration, and is not meant to promote custom add-ons that are not supported by the Home Assistant project.

As they are not supported, we don't want to add these things, as it might sounds like we do.

../Frenck